libs/ui-lib/lib/ocm/services/ClustersService.ts

```diff
@@ -16,6 +16,7 @@ import {
   CustomManifestValues,
   ListManifestsExtended,
 } from '../components/clusterConfiguration/manifestsConfiguration/data/dataTypes';
+import { Buffer } from 'buffer';
```

Comment:
THIS IS A NODEJS MODULE!!! It is not part of the web APIs.

If you are looking to convert a string to base64, why didn't you use `window.btoa()`
See this article for reference: https://developer.mozilla.org/en-US/docs/Glossary/Base64#solution_1_%E2%80%93_escaping_the_string_before_encoding_it